api_app/analyzers_manager/observable_analyzers/maxmind.py

```diff
@@ -69,23 +70,31 @@ def _get_api_key(cls) -> Optional[str]:
             if plugin.value:
                 return plugin.value
         return None
-
+    
+    @classmethod
+    def _get_account_id(cls) -> Optional[str]:
+        for plugin in PluginConfig.objects.filter(
+            parameter__python_module=cls.python_module,
+            parameter__is_secret=True,
+            parameter__name="account_id_name",
+        ):
+            if plugin.value:
+                return plugin.value
+        return None
```

Comment:
this is not necessary. You should do a migration instead. Please read the contribution guidelines that explain how to change an existing analyzer